### DISCUSSION TITLE
Front End development

### DISCUSSION THREAD

**Discussion by Arotiana137-51** [jumperexchange/jumper-exchange#1525] [General]
I found this anouncement on upwork https://www.upwork.com/jobs/~021858476779260875198?referrer_url_path=%2Fbest-matches%2Fdetails%2F~021858476779260875198
So, if this is the repo for this project. I am your man.

**Comment by chybisov** [comment]
@Arotiana137-51 Hey 👋 Thank you for reaching out, but this is not our Upwork posting.
